### ISSUE TITLE
Compress Migration to Sponge Pages

### ISSUE THREAD

**Issue by BlackGoat** [SpongePowered/SpongeDocs#78]
The page users/compat is a launchpad for pages to explain migrating your server from Forge, Bukkit and now Canary, to a Forge server running Sponge mod. These could be compressed into a single page, and we should add mention of Spigot and Cauldron. (Is it conceivable that anyone would migrate from Granite or Glowstone in the future?) Migrating to other implementations of Sponge may be a useful addition later too.
TL;DR Summary of my plan here:
* Combine Bukkit and Canary pages into Forge page, expand to add other servers.
* Keep Compat page (add migrating to Glowstone and Granite pages when appropriate).

**Comment by BlackGoat** [comment]
This suggestion is a consequence of discussion on PR #55 and may be affected by the proposed change of structure @tyrannokapi referenced in issue #86. Unless there are more implementations by then...
